app/workers/send_chase_email_to_referees_worker.rb

```diff
@@ -0,0 +1,13 @@
+class SendChaseEmailToRefereesWorker
+  include Sidekiq::Worker
+
+  def perform
+    GetRefereesToChase.new.perform.each do |reference|
+      begin
+        SendChaseEmail.new.perform(refence: reference)
+      rescue StandardError => e
+        Rails.logger.warn "[DBD] ignoring reference #{reference.id}: #{e.message}"
+      end
```

Comment:
Hmm, why is this needed? 🤔 

Why are we prefixing this with `[DBD]`, looks like a copy+paste typo?

Could we rescue on something more specific than `StandardError`?